server/src/test/java/org/elasticsearch/index/mapper/CompletionFieldMapperTests.java

```diff
@@ -304,6 +306,11 @@ public void testCompletionWithContextAndSubCompletion() throws Exception {
                 contextSuggestField("timmy"),
                 contextSuggestField("starbucks")
             ));
+            try (TokenStream ts = indexableFields.getFields("field.subsuggest")[0].tokenStream(Lucene.WHITESPACE_ANALYZER, null)) {
```

Comment:
In my initial implementation contexts weren't getting transferred correctly to subfields, and iterating over the field tokenstream would throw an exception.  This was picked up by rest tests, but we didn't have a unit test for it.  As it's not easy to check that the field has been correctly built via introspection, I used this as a way to trigger the bug in a unit test so I could track it down more easily.  I'll add a comment, I think it's probably worth keeping here just because it fails faster than a rest test.